### PR TITLE
TASK-7294: TECH_US06: Create Notes rest service

### PR DESCRIPTION
**Pull request by mkrout**

### CODE REVIEW

File: wiki-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java [hunk 1]
```diff
@@ -99,28 +100,8 @@ public NotesRestService(WikiService wikiService, ResourceBundleService resourceB
...
-      return Response.ok(outputText, MediaType.TEXT_HTML).cacheControl(cc).build();
-    } catch (Exception e) {
-      log.error("Error while converting wiki page content: " + e.getMessage(), e);
-      return Response.serverError().entity(e.getMessage()).cacheControl(cc).build();
-    }
-  }
-
   @GET
-  @Path("/{wikiType}/{wikiOwner:.+}/pages/{pageId}")
+  @Path("/pages/{wikiType}/{wikiOwner:.+}/{pageId}")
```
Comment (plamarque) [inline wiki-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java]: should be /notes

Comment (mkrout) [reply wiki-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java]: /notes is the path of the nodes's rest service here we added /pages referring to the data type (we may have other calls for other data types as comments, attachments, wikis...)
So in this case the rest call of this method is ../rest/nodes/pages/{wikiType}/{wikiOwner:.+}/{pageId}

Comment (plamarque) [reply wiki-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java]: then I would prefer /notes/note/

page was the name for wikis. we changed the terminology. Now it’s notebooks for wikis and notes for page

Comment (mkrout) [reply wiki-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java]: Ok, I'll change it

**Pull request closed without merging**